scanpy/external/pl.py

```diff
@@ -14,6 +14,8 @@
     doc_show_save_ax,
 )
 from ..plotting._tools.scatterplots import _wraps_plot_scatter
+from ..plotting import _utils
+from .tl._wishbone import load_plot_helper
```

Comment:
```suggestion
from .tl._wishbone import load_plot_helper as _load_plot_helper
```

You just need to do this to fix the tests. Thank you!